src/sql/swaps/evm.sql

```diff
@@ -49,3 +49,4 @@ JOIN pools      AS p ON p.pool = s.pool
     AND ({protocol:String}   = '' OR p.protocol       = {protocol:String})
 JOIN contracts  AS c0 ON c0.address = p.token0
 JOIN contracts  AS c1 ON c1.address = p.token1
+ORDER BY timestamp DESC
```

Comment:
Isn't the ordering already happening here?

https://github.com/pinax-network/token-api/pull/68/files#diff-65df69b4efb70c35423b3e6fd8730e6146041c3e74815de6ab44664e0e084359L22

Reply:
It's re-shuffling after JOIN. See example here: https://github.com/pinax-network/token-api/issues/67
This "ORDER BY" is very light since we are only sorting "LIMIT" rows.